flocker/node/_loop.py

```diff
@@ -312,6 +312,13 @@ def sleep(self):
         :return _Sleep: an instance of `_Sleep` with a duration
             following an exponential backoff curve.
         """
+        Message.log(
+            message_type=u'flocker:node:_loop:delay',
+            log_level=u'INFO',
+            message=u'Intentionally delaying the next iteration of the '
+                    u'convergence loop to avoid RequestLimitExceeded.',
+            current_wait=self._delay
+        )
```

Comment:
Thanks for catching that! It's actually the other Message.log.

Sorta wish eliot would have just blown up the world rather than just failing...

Maybe we really should add some infrastructure to our acceptance test runner to blow up if there is a traceback in the eliot logs...